code/modules/projectiles/projectile/energy/radiation.dm

```diff
@@ -9,10 +9,11 @@
 	var/radiation_chance = 30
 
 /obj/projectile/energy/radiation/on_hit(atom/target, blocked, pierce_hit)
+	. = ..()
 	if (ishuman(target) && prob(radiation_chance))
 		radiation_pulse(target, max_range = 0, threshold = RAD_FULL_INSULATION)
 
-	..()
+	return BULLET_ACT_HIT
```

Comment:
```suggestion
```
It's already returning this from `. = ..()`

Reply:
Plenty other on_hit()s do the same, so I'm just following the existing convention

Reply:
Those are wrong